opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
M_PI is used in the file, and is defined in math.h on Windows

https://github.com/magnesj/opm-parser/blob/72539e28e1c786ba3d1d6b5a3c7192b68a140afb/opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp#L174